### PR TITLE
clear scannerController to avoid show old scans on open

### LINKED ISSUE #3: Scanned docs rest in Scanner when abort
When I cancel the scanner and re-open the scanner, the previous scans are still in the bottom left corner.

How can I reset the scanner on open?

### PR DESCRIPTION
**Pull request by caspahouzer**
- Rebuild with Xcode 12
- Raised Ti SDK
- clear scannerController to avoid show old scans on open

close #3

### CODE REVIEW

File: titanium-scanner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -278,8 +281,9 @@
 				GCC_WARN_UNUSED_VALUE = NO;
 				GCC_WARN_UNUSED_VARIABLE = NO;
 				INSTALL_PATH = /usr/local/lib;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (hansemannn) [inline titanium-scanner.xcodeproj/project.pbxproj]: You have both iOS 11 and iOS 12 mixed up in the build settings. Please verify both project- and target-build-settings.

File: titanium-scanner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -278,8 +281,9 @@
...
 				LIBRARY_SEARCH_PATHS = "";
+				ONLY_ACTIVE_ARCH = NO;
```
Comment (hansemannn) [inline titanium-scanner.xcodeproj/project.pbxproj:286]: Is this setting really required? I usually don't use it.

File: titanium-scanner.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -214,6 +214,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 24DD6D1B1134B66800162E58 /* titanium.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
```
Comment (hansemannn) [inline titanium-scanner.xcodeproj/project.pbxproj:217]: Is this really required?

File: manifest [hunk 4]
```diff
@@ -2,9 +2,9 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 1.0.0
+version: 1.0.1
```
Comment (hansemannn) [inline manifest]: Bump to 2.0.0 since it's a major change.

**Review by hansemannn — CHANGES_REQUESTED** [review]

File: titanium-scanner.xcodeproj/project.pbxproj (hunk 3 above)
Comment (caspahouzer) [reply titanium-scanner.xcodeproj/project.pbxproj:217]: It was added automatically as I switched to yes and back to no

File: titanium-scanner.xcodeproj/project.pbxproj (hunk 2 above)
Comment (caspahouzer) [reply titanium-scanner.xcodeproj/project.pbxproj:286]: idk what happened. just opened the project with Xcode

**Review by hansemannn — APPROVED** [review]
Looks good! Can you please attach the compiled binary to this PR as well? Then I'll update the releases.

**Comment by caspahouzer** [comment]
[ti.scanner-iphone-2.0.0.zip](https://github.com/hansemannn/titanium-scanner/files/5824312/ti.scanner-iphone-2.0.0.zip)

**Comment by hansemannn** [comment]
Thx! I added macOS Catalyst support as well and released the update

**Pull request merged**